ci/jjb/jobs/pulp-packaging.yaml

```diff
@@ -0,0 +1,65 @@
+# This set of Jenkins jobs creates an installs all the
+# necessary tools to build a repository with the help of Koji, then publishes
+# the results.
+
+- job-template:
+    name: 'pulp-packaging-{release_config}'
+    defaults: ci-workflow-runtest
+    node: 'rhel7-np'
+    properties:
+        - build-ownership
+    scm:
+        - git:
+            url: 'https://github.com/pcreech/pulp-packaging.git'
```

Comment:
Is yours pulp packaging different from this repo?

Reply:
Yes!  I am working on a new pulp-packaging repo that will supercede the python scripts we use now, and will mirror similar efforts by foreman and katello teams.  It isn't 'official' yet, and after a bit of final polish, I will be presenting this to pulp to get it added to pulp.

Reply:
Great thank you